scripts/ansible_v2_3_2_0_1_contrib_inventory_openstack.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python
```

Comment:
Copying this in isn't particularly nice. It feels like tech debt. I don't think it stops the pull request from being merged but I think there should be a separate issue created to address how this is managed.

Reply:
Can you suggest a better alternative?

Reply:
One thought would be to manage it as part of the gating venv job and carry the script in the venv so that it is updated at the same time as the version of ansible.

Reply:
That's a pretty good idea. I like it.

Reply:
Card added: https://rpc-openstack.atlassian.net/browse/RE-1195